var/spack/repos/builtin/packages/self/package.py

```diff
@@ -0,0 +1,49 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Self(CMakePackage):
+    """Spectral Element Library in Fortran."""
+
+    homepage = "https://self.fluidsnumerics.com"
+    url = "https://github.com/FluidNumerics/SELF/archive/refs/tags/v0.0.1.tar.gz"
+
+    maintainers("fluidnumerics_joe", "garrettbyrd")
+
+    license("BSD-3-Clause-Attribution", checked_by="fluidnumerics_joe")
+
+    version("0.0.1", sha256="c3d4ab7d12f18b9d5864c5f9d7a91c86ebc3b7ff0fec4fa56b5c34b325344259")
+
+    variant(
+        "build_type",
+        default="Release",
+        description="The build type to build",
+        values=("Debug", "Release", "Coverage", "Profile"),
+    )
+    variant(
+        "fp64", default=True, description="Enables double precision floating point arithmetic."
+    )
+    variant("examples", default=True, description="Enables build and installation of examples")
+    variant("tests", default=False, description="Enables build and installation of unit tests")
+
+    depends_on("cxx", type="build")
+    depends_on("fortran", type="build")
```

Comment:
```suggestion
    depends_on("c", type="build")
    depends_on("cxx", type="build")
    depends_on("fortran", type="build")
```
See https://github.com/FluidNumerics/SELF/blob/5f6d1e737e579f13d3a753149815e7333da10b5b/CMakeLists.txt#L34